### ISSUE TITLE
Undeclared scala-library 2.11.12 in cross-built multi projects

### ISSUE THREAD

**Issue by barambani** [cb372/sbt-explicit-dependencies#9]
A cross-built multiple project of mine doesn't pass the `undeclaredCompileDependenciesTest` check with the message 
```
[warn] The project depends on the following libraries for compilation but they are not declared in libraryDependencies:
[warn] "org.scala-lang" % "scala-library" % "2.11.12"
```
when compiled with Scala 2.11.12. To reproduce the behaviour in sbt:
```
sbt:log-effect> ++2.11.12
[info] Setting Scala version to 2.11.12 on 4 projects.
[info] Reapplying settings...
[info] Set current project to log-effect (in build file:/Users/fmariotti/open-src/log-effect/)
```
```
sbt:log-effect> undeclaredCompileDependenciesTest
[info] The project explicitly declares all the libraries that it directly depends on for compilation. Good job!
[warn] The project depends on the following libraries for compilation but they are not declared in libraryDependencies:
[warn] "org.scala-lang" % "scala-library" % "2.11.12"
[warn] The project depends on the following libraries for compilation but they are not declared in libraryDependencies:
[warn] "org.scala-lang" % "scala-library" % "2.11.12"
[warn] The project depends on the following libraries for compilation but they are not declared in libraryDependencies:
[warn] "org.scala-lang" % "scala-library" % "2.11.12"
[error] (core / undeclaredCompileDependenciesTest) Failing the build because undeclared dependencies were found
[error] (fs2 / undeclaredCompileDependenciesTest) Failing the build because undeclared dependencies were found
[error] (zio / undeclaredCompileDependenciesTest) Failing the build because undeclared dependencies were found
[error] Total time: 2 s, completed 21-Sep-2018 16:02:44
```
is it something expected (unexpected project configuration) ?
The project's build.sbt is [here](https://github.com/laserdisc-io/log-effect/blob/master/build.sbt#L91)
Thanks

**Comment by cb372** [comment]
Thanks for the report. I've released a fix in v0.2.4.

**Comment by barambani** [comment]
thank you 👍
